lib/metanorma/cli/generator.rb

```diff
@@ -58,6 +59,8 @@ def create_metanorma_document
       end
 
       def find_template(type)
```

Comment:
What about we only update the `custom_template` method to the following?

```ruby
def custom_template 
  template = options[:template]

  if template && template !~ URI::regexp
    Pathname.new(template).join(type)
  elsif template
    Cli::GitTemplate.download(type, repo: template)
  end
end
```

Since `custom_template` is the first, so if that matches then it would not look for find template.

Reply:
Agree.

Reply:
Done